api-reference/_hidden/GridBaseColumn/encodeHtml.md

```diff
@@ -8,4 +8,6 @@ default: true
 Specifies whether HTML tags are displayed as plain text or applied to the values of the column.
 
 ---
-When **true**, HTML tags are displayed as plain text; when **false**, they are applied to the values of the column.
\ No newline at end of file
+When **true**, HTML tags are displayed as plain text; when **false**, they are applied to the values of the column.
+
+HTML tags are always exported to Excel as plain text.
```

Comment:
You should create a hand-made file for the DataGrid because GridBaseColumn is inherited by the TreeList, which doesn't have export.